package.json

```diff
@@ -36,7 +36,7 @@
     "buffer": "^6.0.3",
     "colord": "^2.9.3",
     "dayjs": "^1.11.2",
-    "expo": "49.0.7",
+    "expo": "49.0.13",
```

Comment:
I updated expo and secure-store just to make sure that our Face ID problems are not because of old versions.